source/ed/osm2ed.cpp

```diff
@@ -298,12 +298,14 @@ void OSMCache::insert_ways(){
         values.push_back(std::to_string(way.osm_id));
         values.push_back(way.name);
         values.push_back("way:"+std::to_string(way.osm_id));
+        values.push_back("");
```

Comment:
We may do something a bit smart from `highway` tag here, but it would probably be a bit too long to handle.

Reply:
what do you mean exactly ?
From what I see **OSM** has no information about `type`, before we didn't fill this field, anyway.

Reply:
Yep, it would be an improvement on an other feature, not the point of this PR.
From OSM we could maybe find matching types but that needs spec and a fine OSM work... Not now!